src/app/routes/homePage/getInitialData/index.ts

```diff
@@ -37,7 +36,7 @@ export default async ({
 
     const optHeaders = { 'ctx-service-env': env };
 
-    // @ts-ignore - Ignore fetchPageData argument types
+    // @ts-expect-error - Ignore fetchPageData argument types
```

Comment:
We can't always fix them... we have chosen to use `ts-expect-error` over `ts-ignore` (see https://paper.dropbox.com/doc/Migrate-MostRead-from-Legacy--B43g~QNaxpavx9j5UKF1MoreAg-0FHywUTsf55Brd9VaYUxs#:uid=470449461357824298478247&h2=Use-@ts-expect-error-instead-o), because you need to supply a reason. 

If something that is annotated with `ts-expect-error` gets fixed by something else, and it's not an error anymore, then type-checking will kick in and force us to remove.

Ideally we should not have a lot of `ts-expect-errors` in our production code, it is more often used in tests for error handling, or testing with partial data / incomplete props etc. 

But that said, there would be no harm in having some type of ticket in our tech backlog to come and assess / revisit / audit these every once in a while?

Reply:
Awesome, good work with using `ts-expect-error` sounds ideal!